src/components/Layer2ProductCard.tsx

```diff
@@ -1,12 +1,12 @@
-// Libraries
 import { StaticImageData } from "next/image"
+import Image from "next/image"
```

Comment:
Use `TwImage` in https://github.com/kushagrasarathe/ethereum-org-website/blob/dev/src/components/Image.tsx#L52